File: richest-customer-wealth/richest-customer-wealth.go
func maximumWealth(accounts [][]int) int {
    MaxWealth := 0
    for i :=0; i< len(accounts); i++{
        CurrentWealth := 0
        for j :=0; j < len(accounts[i]); j++{
            CurrentWealth += accounts[i][j]
        }
           if CurrentWealth > MaxWealth{
        MaxWealth = CurrentWealth
    } 
    }
    return MaxWealth
}

File: richest-customer-wealth/richest-customer-wealth.py
class Solution(object):
    def maximumWealth(self, accounts):
        """
        :type accounts: List[List[int]]
        :rtype: int
        """
        maxwealth = 0
        for i in range(len(accounts)):
            totalwealth = sum(accounts[i])
            maxwealth = max(maxwealth, totalwealth)
        return maxwealth